index.bs

```diff
@@ -77,16 +88,16 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The magnetometer's associated {{Sensor}} subclass is the {{Magnetometer}} class.
-
-The Magnetometer has a <a>default sensor</a>, which is the device's main magnetomer sensor.
+The magnetometer has two associated {{Sensor}} subclasses, {{Magnetometer}} and {{RawMagnetometer}}.
 
-The Magnetometer has an associated {{PermissionName}} which is <a for="PermissionName" enum-value>"magnetometer"</a>.
+The {{Magnetometer}} and {{RawMagnetometer}} have an associated {{PermissionName}} which is <a for="PermissionName" enum-value>"magnetometer"</a>.
 
-A [=latest reading=] per [=sensor=] of magnetometer type includes three [=map/entries=]
-whose [=map/keys=] are "x", "y", "z" and whose [=map/values=] contain current <a>magnetic field</a>
+For {{Magnetometer}} class, the [=latest reading=] includes three [=map/entries=]
```

Comment:
The way generic sensor is spec'ed, latest-reading is tied to the sensor concept, and not to interfaces.